util/test/test_dftool.py

```diff
@@ -118,26 +118,8 @@ def test_manyrandom():
 def test_noarg_commands():
     # Test commands that have no args. They're usually help commands;
     # sometimes they do the main work (eg compile).
-    argv1s = [
-        "",
-        "query",
-        "volsym",
-        "getrate",
-        "calc",
-        "dispense",
-        "dispense_active",
-        "querymany",
-        "compile",
-        "manyrandom",
-        "newdfrewards",
-        "mine",
-        "newacct",
-        "newtoken",
-        "acctinfo",
-        "chaininfo",
-    ]
+    argv1s = ["", "compile", "newacct"]
```

Comment:
In line with previous comment: do _not_ add this line.

Reply:
Reverted to the full list and removed query/dispense.